Conversion/MultiCopy.FCMacro

```diff
@@ -0,0 +1,1439 @@
+    result = ""
+    try:
+        for numeral, integer in romanNumeralMap:
+            while n_ >= integer:
+                result += numeral
+                n_ -= integer
+    except Exception:
+        result = ""
+    return result
+
+def RomanToInt(inputStr_):
+    result = 0
+    index = 0
+    try:
+        for numeral, integer in romanNumeralMap:
+            while inputStr_[index : index + len(numeral)] == numeral:
+                result += integer
+                index += len(numeral)
+    except Exception:
+        result = 0
+    return result
+
+
+
+# Alphabet Conversion
+# ------------------------------------------------------------------------------------------------
+
+alphaMap = [
+    RANDOMSTRING,
+    "a",
+    "b",
+    "c",
+    "d",
+    "e",
+    "f",
+    "g",
+    "h",
+    "i",
+    "j",
+    "k",
+    "l",
+    "m",
+    "n",
+    "o",
+    "p",
+    "q",
+    "r",
+    "s",
+    "t",
+    "u",
+    "v",
+    "w",
+    "x",
+    "y",
+    "z",
+]
+
+def IntToAlpha(n_):
+    result = ""
+    try:
+        if n_ >= 1 and n_ <= 26:
+            result = alphaMap[n_]
+        elif n_ >= 27 and n_ <= 702:
+            result = (
+                alphaMap[math.ceil((n_ - 26) / 26)]
+                + alphaMap[n_ - (26 * math.ceil((n_ - 26) / 26))]
+            )
+        elif n_ >= 703 and n_ <= 18278:
+            tempVar1 = math.ceil((n_ - 702) / 26) % 26
+            tempVar2 = math.ceil(n_ - 702) % 26
+            tempVar1 = 26 if tempVar1 == 0 else tempVar1
+            tempVar2 = 26 if tempVar2 == 0 else tempVar2
+            result = (
+                alphaMap[math.ceil((n_ - 702) / 676)]
+                + alphaMap[tempVar1]
+                + alphaMap[tempVar2]
+            )
+    except Exception:
+        result = ""
+    return result
+
+def AlphaToInt(inputStr_):
+    result = 0
+    try:
+        if len(inputStr_) == 1:
+            result = alphaMap.index(inputStr_)
+        elif len(inputStr_) == 2:
+            result = (26 * alphaMap.index(inputStr_[0])) + alphaMap.index(inputStr_[1])
+        elif len(inputStr_) == 3:
+            result = (
+                (676 * alphaMap.index(inputStr_[1]))
+                + (26 * alphaMap.index(inputStr_[1]))
+                + alphaMap.index(inputStr_[2])
+            )
+    except Exception:
+        result = 0
+    return result
+
+
+
+###################################################################################################################
+###-------------------------------------------------------------------------------------------------------------###
+### 					MULTICOPY MACRO CALLS							###
+###-------------------------------------------------------------------------------------------------------------###
+###						  		   						###
+###								   						###
+###	This is the main macro call. The code below commences 	   						###
+###	the MultiCopy GUI interface. This script cannot be 	   						###
+###	called externally.					   						###
```

Comment:
If the code cannot be called externally, you should remove the first line (`#!...`).

Reply:
Done! I kept the first line because it highlights the code in the text editor, and also it shows the Python file icon in the directory.

Reply:
I guess you have a way to persuade your GUI and desktop environment that a FCMacro file is a Python file.

Reply:
Funnily, I think I do.